src/Core/Commander/Providers/WMTS_Provider.js

```diff
@@ -109,147 +73,65 @@ WMTS_Provider.prototype.url = function url(coWMTS, layer) {
  * @param {type} coWMTS : coord WMTS
  * @returns {WMTS_Provider_L15.WMTS_Provider.prototype@pro;_IoDriver@call;read@call;then}
  */
-WMTS_Provider.prototype.getXbilTexture = function getXbilTexture(tile, layer, parameters) {
-    var cooWMTS = tile.wmtsCoords[layer.options.tileMatrixSet][0];
-    var pitch = new THREE.Vector3(0.0, 0.0, 1.0);
-
-    if (parameters.ancestor) {
-        // account for possible level offset between coords and tile level
-        // (e.g for PM texture cooWMTS.level = tile.level + 1)
-        var levelOffset = cooWMTS.zoom - tile.level;
-
-        cooWMTS = this.projection.WMTS_WGS84Parent(
-            cooWMTS,
-            this.computeLevelToDownload(tile, parameters.ancestor, layer) + levelOffset,
-            pitch);
-    }
-
-    var url = this.url(cooWMTS, layer);
-
-    // TODO: this is not optimal: if called again before the IoDriver resolves, it'll load the XBIL again
-    var textureCache = this.cache.getRessource(url);
-
-    if (textureCache !== undefined) {
-        const minmax = this._IoDriver.computeMinMaxElevation(
-            textureCache.floatArray,
-            SIZE_TEXTURE_TILE, SIZE_TEXTURE_TILE,
+WMTS_Provider.prototype.getXbilTexture = function getXbilTexture(tile, layer, parentTextures) {
+    let coordWMTS = tile.wmtsCoords[layer.options.tileMatrixSet][0];
+    const pitch = new THREE.Vector3(0.0, 0.0, 1.0);
+
+    if (parentTextures) {
+        coordWMTS = Tish.WMTS_WGS84Parent(
```

Comment:
This only works with `WGS84G` tileMatrixSet right? (As you assume there's only one texture)
If this is right, add throw an error when this condition is false.